tree/30_generic_methods/service_action.cf

```diff
@@ -47,7 +47,17 @@ bundle agent service_action(service_name, action)
       "svc_action_cmd[stop]"      string => "/usr/bin/stopsrc -s ${service_name}";
       "action_command"            string => "${svc_action_cmd[${action}]}";
 
-    !systemctl_utility_present.!service_utility_present.!startsrc_utility_present.init_d_directory_present::
+    !systemctl_utility_present.!service_utility_present.!startsrc_utility_present.svcadm_utility_present::
+      "svc_action_cmd[restart]"   string => "${paths.path[svcadm]} restart -s ${service_name}";
+      "svc_action_cmd[refresh]"   string => "${paths.path[svcadm]} refresh -s ${service_name}";
+      "svc_action_cmd[reload]"    string => "${svc_action_cmd[refresh]}";
+      "svc_action_cmd[enable]"    string => "${paths.path[svcadm]} enable -s ${service_name}";
+      "svc_action_cmd[start]"     string => "${svc_action_cmd[enable]}";
```

Comment:
I think we should add -t option to start and to stop.
If the -t option is specified, svcadm temporarily enables each service instance.
ie this is the difference between start and enable